### PR TITLE
Fix image links and rest refman

### LINKED ISSUE #1114: BUG-Missing images in Extension documentation
**Bug description**
There are missing images in the Extension documentation.

image

https://docs.taipy.io/en/develop/userman/gui/extension/dynamic_element/scalar_props/#using-the-element-in-the-application

### PR DESCRIPTION
**Pull request by FlorianJacta**
Resolves: #1114 

There was an error in how the Rest Refman was generated. I fixed the issue by not using the absolute path.

image

image

### CODE REVIEW

File: tools/_setup_generation/step_rest_refman.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def get_property_type(property_desc, from_schemas):
                 if dest_path in enterprise_paths:
                     file.write(Setup.ENTERPRISE_BANNER)
                 file.write(f"{dest_path_desc}\n\n")
-                self.navigation += f"- \"Paths for {dest_path}\": {REST_REF_DIR_PATH}/apis_{dest_path}.md\n"
+                self.navigation += f'- "Paths for {dest_path}": refmans/reference_rest/apis_{dest_path}.md\n'
```
Comment (jrobinAV) [inline tools/_setup_generation/step_rest_refman.py:103]: Can't you use the REST_REF_DIR_PATH variable? What is the issue?

File: tools/_setup_generation/step_rest_refman.py [hunk 2]
```diff
@@ -122,10 +122,16 @@ def get_property_type(property_desc, from_schemas):
...
+                        f'<div><h4 style="display: inline-block;">Request body:</h4>\n'
+                    )
+                    schema = request_body["content"]["application/json"]["schema"][
+                        "$ref"
+                    ]
                     schema = schema[schema.rfind("/") + 1 :]
-                    file.write(f'<span><a href="schemas.md#{schema.lower()}">{schema}</a></div>\n\n')
+                    file.write(
+                        f'<span><a href="schemas.md#{schema.lower()}">{schema}</a></div>\n\n'
+                    )
```
Comment (jrobinAV) [inline tools/_setup_generation/step_rest_refman.py:134]: I believe your automatic formatting is incorrect. Can you check?

**Review by jrobinAV — APPROVED** [review]

**Review by FabienLelaquais — APPROVED** [review]
👌

File: tools/_setup_generation/step_rest_refman.py (hunk 1 above)
Comment (FlorianJacta) [reply tools/_setup_generation/step_rest_refman.py:103]: This is the error. I cannot use REST_REF_DIR_PATH as it is the absolute path to the file and the goal here is to use the relative path for the navigation

File: tools/_setup_generation/step_rest_refman.py (hunk 2 above)
Comment (FlorianJacta) [reply tools/_setup_generation/step_rest_refman.py:134]: I am using Black to format the code. What do I have to do?

**Pull request merged**